### ISSUE TITLE
Could you please lambda saving last deployed url from s3?

### ISSUE THREAD

**Issue by nikita-sheremet-java-developer** [aws/aws-lambda-java-libs#31]
Could you please make lambda remeber last deployed url from s3? It is very annoying to copy and paste it every time when I need to redeploy lambda.

![image](https://user-images.githubusercontent.com/33242225/32415232-e414b65a-c268-11e7-8c1c-b939e4a7ff4f.png)

It is may be out of your scope, but I have no idea where to write this. Amazon it sefl does not have any feature request mechanism (why?), so if this is not of your scope could you please pass it to right place?

**Comment by asuresh8** [comment]
This is out of scope but I will forward it to the right team! Also, I'll try to find out the correct location for AWS console requests to go to.

**Comment by asuresh8** [comment]
This feature is now being tracked internally. Feel free to submit future console requests here, and we'll make sure they get sent to the right team.

I'm going to close out this issue as it's not something our team can track, but I can confirm that it's being worked on!

**Comment by nikita-sheremet-java-developer** [comment]
Thank you. But why does amazon team not share issue tracker for public usage? Look at JetBrains - every user (no matter has he/she an Intellij product or not) can register and add bug or feature request. I **understand** that amazon is **not** open source project. And your director/manager may tell that issues is a kind of support and should be paid. But again look at JetBrains, they dispatch issue (e.g. mark it as feature requerts or bug, or payment request etc.) and only after that start to work. E.g. if requested issue is a kind of support you can redirect it to user and hide it from others, but if it is a feature request or bug you can work on it.

Also if we speak about feature requests you can add voting to them to see how popular it is. Because for now it is really unclear where to send message/email. For example aws cosole have a nice UI bug, but I have no idea where to send it:

![image](https://user-images.githubusercontent.com/33242225/32530616-c30f4082-c471-11e7-8521-e812a4a9f4f4.png)

P.S. about JetBrains it is **not** an advertisement it is just an obvious example. Actually many of compaines do that.
